packages/card/src/CardTitle.js

```diff
@@ -24,7 +24,9 @@ import CardRow from './CardRow';
 import cardTheme from './cardTheme';
 
 type Props = {|
-  /** Rendered content of the component */
+  /** Category title displayed above the title */
+  categoryTitle?: string,
```

Comment:
@brentertz, @coldpour — Curious for your thoughts on this alternative approach: https://github.com/mineral-ui/mineral-ui/pull/187/commits/a4d60be4a97b92aadbd2a39c0005e3838f782c93